packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -639,6 +639,8 @@ export const messages = {
     'Press Enter to confirm your input or Escape to cancel',
   rename_component_warning:
     'Warning: References to the old name will not be updated. Update manually and redeploy once all changes have been made.',
+  rename_component_error:
+    'Unable to rename component. Update manually and redeploy once all changes have been made.',
```

Comment:
@sbudhirajadoc - I added a pop-tart message if there is ever an error running this command.  I noticed that we also throw a generic failure message.  So this might not be necessary.  Let me know what you think.  

The nasty details of the error message will display in the output channel.
<img width="503" alt="Screen Shot 2022-06-30 at 2 31 33 PM" src="https://user-images.githubusercontent.com/599418/176780318-16bcc655-a3cc-48ac-ab86-e906f415b964.png">

Reply:
@dehru I am not clear on what the action "Update manually and redeploy once all the changes have been made" means. What do they update? How  do they redeploy?